src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemExtensionsTests.cs

```diff
@@ -225,6 +225,94 @@ public void SarifWorkItemExtensions_GetRunToolNames_FetchesAllRunToolNames()
             toolNames.Count.Should().Be(0);
         }
 
+        [Fact]
+        public void SarifWorkItemExtensions_CreateWorkItemDescription_SingleResultMultipleLocations()
+        {
+            string toolName = "TestToolName";
+            string firstLocation = @"C:\Test\Data\File1.sarif";
+            string additionLocationCount = "2";
+
+            SarifLog sarifLog = TestData.CreateOneIdThreeLocations();
+            sarifLog.Runs[0].VersionControlProvenance = null;
+
+            string description = sarifLog.CreateWorkItemDescription(new SarifWorkItemContext() { CurrentProvider = Microsoft.WorkItems.FilingClient.SourceControlProvider.AzureDevOps });
+
+            description.Should().BeEquivalentTo($"This work item contains 1 '{toolName}' issue(s) detected in {firstLocation} (+{additionLocationCount} locations). Click the 'Scans' tab to review results.");
+        }
+
+        [Fact]
+        public void SarifWorkItemExtensions_CreateWorkItemDescription_SingleResultWithMultipleArtifacts()
```

Comment:
>SarifWorkItemExtensions_CreateWorkItemDescription_SingleResultWithMultipleArtifacts [](http://example.com/codeflow?start=20&length=83)

Let's also create two more tests:
1. multiple results and count the number of issues
2. one result with multiple locations